bitnami/grafana-loki/templates/compactor/service.yaml

```
@@ -48,6 +48,15 @@ spec:
       {{- else if eq .Values.compactor.service.type "ClusterIP" }}
       nodePort: null
       {{- end }}
+    - name: grpc
+      port: {{ .Values.compactor.service.ports.grpc }}
+      targetPort: http
```

Shouldn't it be `grpc` ?

```suggestion
      targetPort: grpc
```